android/src/main/java/com/brentvatne/exoplayer/FullScreenPlayerView.kt

```diff
@@ -60,72 +58,41 @@ class FullScreenPlayerView(
     init {
         setContentView(containerView, generateDefaultLayoutParams())
     }
+
     override fun onBackPressed() {
-        super.onBackPressed()
+        findViewById<ImageView>(androidx.media3.ui.R.id.exo_fullscreen)?.performClick()
```

Comment:
Do we really need this code?

Reply:
Yeah, we need it because when a user presses the back button, the full-screen button icon doesn't change. We have to add it to update the full-screen button icon.